frontend/src/scenes/trends/viz/ActionsPie.tsx

```diff
@@ -74,8 +74,8 @@ export function ActionsPie({
                             const { dataset } = point
                             const action = dataset.actions[point.index]
                             const label = dataset.labels[point.index]
-                            const date_from = dataset.days[0]
-                            const date_to = dataset.days[dataset.days.length - 1]
+                            const date_from = filtersParam.date_from || ''
```

Comment:
non-blocking

What are the cases where `loadPeople` would use the explicitly provided `date_from` and `date_to` over using whatever `filterParams` already provides?

How might I know that `''` is going to do what's expected?

Reply:
loadPeople differs between 2 modes. One a line graph when you're selecting a specific date, you need to specify the exact timeframe among a date range. On a total chart (like pie chart or bar chart by value), you should just provide the filters default and not set it to anything specifically.

> How might I know that '' is going to do what's expected?

This is where the typing you're working on could help. It should make explicit what should always be passed/not passed and what defaults are allowed + what the defaults do. The empty string defers the date control to whatever default we have set up within the date parsing functions when query building